homeassistant/components/eight_sleep/binary_sensor.py

```diff
@@ -49,13 +42,15 @@ class EightHeatSensor(EightSleepBaseEntity, BinarySensorEntity):
 
     def __init__(
         self,
+        entry: ConfigEntry,
         coordinator: DataUpdateCoordinator,
         eight: EightSleep,
         user_id: str | None,
         sensor: str,
     ) -> None:
         """Initialize the sensor."""
-        super().__init__(coordinator, eight, user_id, sensor)
+        super().__init__(entry, coordinator, eight, user_id, sensor)
+        self._attr_device_class = BinarySensorDeviceClass.OCCUPANCY
```

Comment:
You can move this above init as `_attr_device_class = BinarySensorDeviceClass.OCCUPANCY` since its static

Reply:
it's a dupe